client-js/app/elements/labrad-registry.ts

```diff
@@ -268,6 +363,24 @@ export class LabradRegistry extends polymer.Base {
     }
   }
 
+  doDragCopy() {
+    var self = this;
+    var newName =  this.$.dragCopyNameInput.value;
+    var newPath = JSON.parse(this.$.dragCopyPathInput.value);
+    var oldPath = JSON.parse(this.$.originPath.textContent);
+    var oldName = this.$.originName.textContent;
+
+    this.socket.copyDir({path: oldPath, dir: oldName, newPath: newPath, newDir: newName}).then(
+      (resp) => {
+        self.repopulateList(resp);
```

Comment:
You'll want to change this to match the other api calls after rebasing, since `copyDir` no longer returns a directory listing.